core/domain/blog_domain_test.py

```diff
@@ -315,6 +315,25 @@ def test_valid_thumbnail_update(self) -> None:
             'Expected image filename to be a string, received 123'
         )
 
+    def _assert_valid_tag_elements(
+        self, expected_error_substring: str
+    ) -> None:
+        """Checks that blog post passes validation for tags."""
```

Comment:
It looks like this function actually asserts that the tags are invalid, not that they are valid